modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/metadata/MetadataOutputDelegate.java

```diff
@@ -175,6 +181,9 @@
           .withReason(NULL_TYPE_ERROR)
           .onOutputAttributes();
       return failure(failure);
+    } catch (ConnectionException e) {
+      return failure(newFailure(e).withMessage("Failed to establish connection: " + ExceptionUtils.getMessage(e))
+          .withFailureCode(CONNECTION_FAILURE).onKeys());
```

Comment:
and here

Reply:
Good catch. Yes, it should be onComponent().